profiler/src/main/java/com/splunk/opentelemetry/profiler/Configuration.java

```diff
@@ -59,4 +60,12 @@ public static String getConfigUrl(Config config) {
     }
     return ingestUrl;
   }
+
+  public static boolean getTLABEnabled(Config config) {
```

Comment:
Why static? Having this as instance method will simplify `ConfigurationLogger` as well

Reply:
Implemented it the same way as `getConfigUrl` for consistency. Should I change both?

Reply:
It certainly does not make sense to me to use a static method and to pass as a parameter an instance of this very same class

Reply:
It's easy to miss the fact that `Configuration` is not `Config`. This is a convenience method. Because it's confusing, maybe if the call sites are limited we can just move it closer to the call site(s).

Reply:
Would not really call this a convenience method since it contains logic that the callsite should not need to know about (which configuration option inherits its value from which). A convenience method by definition should exist for the sole reason of convenience, not encapsulation as this method does.

Reply:
> It's easy to miss the fact that `Configuration` is not `Config`. This is a convenience method. Because it's confusing, maybe if the call sites are limited we can just move it closer to the call site(s).

Ou, I totally missed that